docs/_docs/2_4_configurations.md

```diff
@@ -396,56 +396,90 @@ Property: `hoodie.compaction.daybased.target` <br/>
 Property: `hoodie.compaction.payload.class` <br/>
 <span style="color:grey">This needs to be same as class used during insert/upserts. Just like writing, compaction also uses the record payload class to merge records in the log against each other, merge again with the base file and produce the final record to be written after compaction.</span>
 
-
 ### Metrics configs
+
 Enables reporting on Hudi metrics.
 [withMetricsConfig](#withMetricsConfig) (HoodieMetricsConfig) <br/>
-<span style="color:grey">Hudi publishes metrics on every commit, clean, rollback etc.The following sections list the supported reporters.</span>
+<span style="color:grey">Hudi publishes metrics on every commit, clean, rollback etc. The following sections list the supported reporters.</span>
 
 #### GRAPHITE
 
-##### on(metricsOn = true) {#on}
+##### withMetricsOn(metricsOn = true) {#withMetricsOn}
```

Comment:
Trying to make `with` the same prefix for this builder pattern. IMO having different prepositions is just confusing.